docs/designers-developers/developers/block-api/README.md

```diff
@@ -4,7 +4,7 @@ Blocks are the fundamental element of the Gutenberg editor. They are the primary
 
 ## Registering a block
 
-All blocks must be registered before they can be used in the editor. You can learn about block registration, and the available options, in the [block registration](block-api/block-registration.md) documentation.
+All blocks must be registered before they can be used in the editor. You can learn about block registration, and the available options, in the [block registration](block-api/block-registration) documentation.
```

Comment:
So our parser will actually correctly handle the `.md`, the problem is actually that it only works if it detects a relative path to `docs/` in the URL. Here's how it would need to look, so it can catch the URL correctly:

```suggestion
All blocks must be registered before they can be used in the editor. You can learn about block registration, and the available options, in the [block registration](../../../../docs/designers-developers/developers/block-api/block-registration) documentation.
```

Reply:
@martinDolan can you apply the change shared by @chrisvanpatten so we could merge this PR?